apple/internal/apple_framework_import.bzl

```diff
@@ -109,7 +109,7 @@ def _apple_framework_import_impl(ctx):
             )
     else:
         if ctx.attr.sdk_dylibs:
-            objc_provider_fields["sdk_dylib"] = ctx.attr.sdk_dylibs
+            objc_provider_fields["sdk_dylib"] = depset(direct = ctx.attr.sdk_dylibs)
```

Comment:
Assuming `direct` is the right option here, but happy to update as needed

Reply:
please do the same below for `sdk_framework` and `weak_sdk_framework`

Reply:
Done!